lib/reports/subscription_changes_after_switch_to_daily_digest_report.rb

```diff
@@ -1,6 +1,13 @@
 class Reports::SubscriptionChangesAfterSwitchToDailyDigestReport
   def self.call
-    updated_subscriptions = Subscription.where(source: 4).where.not(ended_at: nil)
+    experiment_2_list_ids = SubscriberList.where(
+      slug: File.readlines(Rails.root.join("lib/experiment_2_slugs")),
+    )
+
+    updated_subscriptions = Subscription
+      .where(source: 4)
```

Comment:
minor, you can make this a bit more readable with either Subscription.where(source: :bulk_immediate_to_digest)` or even `Subscription.source_bulk_immediate_to_digest`

Reply:
Oh yeah - for some reason originally this didn't work. Updated.